
  teach how to handle orig_pair ESC sequence 
  generated by "TERM=cons25 tput op"
   TERM=cons25 tput op | od -ax
  see #559364

--- a/sys/dev/syscons/teken/sequences
+++ b/sys/dev/syscons/teken/sequences
@@ -88,6 +88,7 @@
 IL	Insert line				^[ [ L		n
 IND	Index					^[ D
 NEL	Next line				^[ E
+OP	Orig Pair				^[ [ x
 RI	Reverse index				^[ M
 RIS	Reset to Initial State			^[ c
 RM	Reset Mode				^[ [ l		r
--- a/sys/dev/syscons/teken/teken_subr.h
+++ b/sys/dev/syscons/teken/teken_subr.h
@@ -1056,6 +1056,15 @@
 }
 
 static void
+teken_subr_orig_pair(teken_t *t)
+{
+	/* Set default foreground color. */
+	t->t_curattr.ta_fgcolor = t->t_defattr.ta_fgcolor;
+	/* Set default background color. */
+	t->t_curattr.ta_bgcolor = t->t_defattr.ta_bgcolor;
+}
+
+static void
 teken_subr_set_graphic_rendition(teken_t *t, unsigned int ncmds,
     unsigned int cmds[])
 {
